### PR TITLE
SELV3-573: add data element mapping table

### PR DESCRIPTION
**Pull request by mdulko**

### CODE REVIEW

File: src/admin-dhis2/components/AdminDhis2DataElementsPage/AdminDhis2DataElementsPage.jsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        []
+    );
+
+    useEffect(() => {
+        const serverItem = JSON.parse(localStorage.getItem('stateLocation'));
+        const datasetItem = JSON.parse(localStorage.getItem('datasetState'));
+
+        if (serverItem !== undefined || datasetItem !== undefined) {
+            setServerId(serverItem.data.serverId);
+            setDatasetId(datasetItem.data.datasetId)
```
Comment (sniedzielski) [inline src/admin-dhis2/components/AdminDhis2DataElementsPage/AdminDhis2DataElementsPage.jsx:44]: missing semicolon

File: src/admin-dhis2/components/AdminDhis2DatasetPage/AdminDhis2DatasetPage.jsx [hunk 2]
```diff
@@ -85,6 +86,19 @@ const AdminDhis2DatasetPage = () => {
         toggleAddModal();
     };
 
+    const goToDataElementsPage = (data) => {
+        const datasetState = {
+            data: data,
+        };
+        const serverItem = JSON.parse(localStorage.getItem('stateLocation'))
```
Comment (sniedzielski) [inline src/admin-dhis2/components/AdminDhis2DatasetPage/AdminDhis2DatasetPage.jsx:93]: missing semicolon

**Review by sniedzielski — COMMENTED** [review]
I left two comments. Please refer to them.

**Review by dariapionk — APPROVED** [review]

File: src/admin-dhis2/components/AdminDhis2DataElementsPage/AdminDhis2DataElementsPage.jsx (hunk 1 above)
Comment (mdulko) [reply src/admin-dhis2/components/AdminDhis2DataElementsPage/AdminDhis2DataElementsPage.jsx]: updated

File: src/admin-dhis2/components/AdminDhis2DatasetPage/AdminDhis2DatasetPage.jsx (hunk 2 above)
Comment (mdulko) [reply src/admin-dhis2/components/AdminDhis2DatasetPage/AdminDhis2DatasetPage.jsx]: updated

**Review by sniedzielski — APPROVED** [review]
LGTM

**Pull request merged**